Formula/pyqt.rb

```diff
@@ -24,19 +24,20 @@ class Pyqt < Formula
 
   def install
     python = Formula["python@3.9"]
+    qt = Formula["qt"]
     args = %W[
       --target-dir #{prefix}
       --no-make
       --confirm-license
     ]
     system "sip-build", *args
     cd "build" do
-      qt_prefix = Formula["qt"].prefix Formula["qt"].version
+      qt_prefix = qt.prefix qt.version
```

Comment:
I'm confused about this line. What happens when you do this?

```suggestion
      qt_prefix = qt.prefix
```

Reply:
It seems to me that in both cases, `qt_prefix == /usr/local/Cellar/qt/6.0.2` (or maybe more generally, `HOMEBREW_CELLAR/qt/6.0.2`).

Reply:
if without the version method [`prefix`](https://rubydoc.brew.sh/Formula.html#prefix-instance_method) returns the `opt_prefix`

Reply:
It seems that's inaccurate. I modified `hello.rb`'s install method to
```ruby
  def install
    ohai Formula["qt"].prefix
  end
```
Then:
```
❯ brew install -s hello
==> Downloading https://ftp.gnu.org/gnu/hello/hello-2.10.tar.gz
Already downloaded: /Users/chisquared/Library/Caches/Homebrew/downloads/0d3dc7a1e45d7ac6eb8d853c8ecccad6a4ab6abbbf38038f552e4735ad15d3cc--hello-2.10.tar.gz
==> /usr/local/Cellar/qt/6.0.2
Error: Empty installation
```

Reply:
Huh, weird. And that's a very weird workaround. Ok. Can we then be more explicit about what we want here? Maybe something like
```ruby
qt_prefix = HOMEBREW_CELLAR/"qt"/qt.version
```